Resources/Locale/en-US/criminal-records/criminal-records.ftl

```diff
@@ -13,6 +13,7 @@ criminal-records-status-detained = Detained
 criminal-records-status-suspected = Suspect
 criminal-records-status-discharged = Discharged
 criminal-records-status-paroled = Paroled
+criminal-records-status-subpoenaed = Subpoenaed
```

Comment:
shouldn't this be under the part where you commented `## DeltaV - Justice Department Subpoena`?

Reply:
One of these days I'll learn. x_x changes made and commtited.